db/seeds.rb

```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.find_each do |bundle_instance|
+  bundle_instance.applications.each do |app|
```

Comment:
Isn't as big of a deal here, as I can't imagine there being more than 1,000 results. But I think it good practice to use it.